CHANGELOG.md

```diff
@@ -0,0 +1,71 @@
+please follow `readme/linux.md` for new, updated ways to install Xournal++.
+Users that installed from package repositories can ignore this notice.
+
+We now officially support packaging Xournal++ using CMake. This allows users to
+generate DEB and tar packages. See `readme/linux.md` for more details.
+
+* **Breaking change**: the desktop and icon files were renamed from `xournalpp`
+  to `com.github.xournalpp.xournalpp` to be more in line with the AppStream
+  specification. This change should only affect users that install with `make
+  install`; these users must run `make uninstall` with _Xournal++ 1.0.12_ first.
+  We recommend migrating to another installation format, such as Flatpak,
+  AppImage, or tarball. Refer to [this
+  issue](https://github.com/xournalpp/xournalpp/issues/1442#issuecomment-524566511)
+  for more details on how to uninstall if `make uninstall` is unavailable.
+* Text field
+    * (New input system) double/triple text selection in text fields
+    * Fixed issue with cursor not showing on mouse movement after typing into
+      text field
+    * Fixed text not rendering correctly when used with highlighter and
+      non-white backgrounds
+* Latex tool
+    * Now warns users if latex dependencies are missing
+    * Newly created objects are now placed in the center of the screen or page
+* Image tool
+    * Images are now selected by default on insertion
+    * Fixed memory leak
+* Other misc features and bugfixes
+    * Added appdata file
+    * Added ability to customize UI styles using CSS
+    * Added a right-click context menu to the page preview sidebar
+    * Added option to disable scrollbar fade
+    * Allow `*.pdf.xopp` filenames. Any file whose name follows the format
+      `$filename.pdf.xopp` will be exported as `$filename.pdf`.
+    * Renamed "Thin" thickness to "Fine"; Added "Very Fine" and "Very Thick"
+      thicknesses
+    * Xournal++ icon is now rendered in thumbnails of Xournal++ files
+    * Added ability for installation to be relocatable
+    * Updated translations
+    * Updated dependency information for DEB packages
+    * Improved copy-paste behavior
+    * Improved file size by reducing stroke coordinate precision
+    * Improved audio recording quality and stability
+    * Enable Enter/Shift+Enter to advance search bar
+    * Enabled left and right arrow keys to change pages in presentation mode
+    * Fixed an issue where discarding changes to current file and then selecting
+      "Cancel" in the file open dialog caused subsequent saves to fail
+    * Fixed an issue where last eraser thickness settings were not being remembered
+    * Fixed pen strokes not appearing after searching
+    * Fixed user-defined toolbar shrinking in size after customizing
+    * Fixed scrollbar issues caused by window resizes
+    * Fixed an issue where pasted images were zoomed in and ignored DPI settings
+    * (Windows) Fixed the help document (actually User Guide) webpage not
+      opening
```

Comment:
add a line (re: 1405, 1477):  
Fixed issues causing pages to flash loading image